.github/workflows/benchmark.yml

```diff
@@ -35,6 +35,5 @@ jobs:
           external-data-json-path: ./benchmarks/data.json
           github-token: ${{ secrets.GITHUB_TOKEN }}
           gh-pages-branch: benchmarks
-          auto-push: true
-          fail-on-alert: false
+          fail-on-alert: true
```

Comment:
This was set to `false` back when we were still running the benchmarks only on a push to main. Now that we run them less frequently, failing them if they alert may make more sense.